docs/README.md

```diff
@@ -437,6 +437,11 @@ Set it to an empty string there to check via `=`/`!=` instead of `IS NULL`/`IS N
 - `mode` (`string`, defaults to `OR`) The conditional mode to use when matching
   against multiple fields. Valid values are `OR` and `AND`.
 
+- `not` (`string`, defaults to `null`) An alternative to `multiValue`, especially if you have
```

Comment:
More than a single character to denote negation doesn't seem necessary. So an option named `negationChar` would be more intuitive.